.gitignore

```diff
@@ -31,3 +31,5 @@ man/man1
 man/man5
 man/man8
 vendor/pkg/
+# generated by integration-cli-on-swarm
```

Comment:
not sure where it is generated, but github complains that there is no last line break